payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/TextFieldController.kt

```diff
@@ -144,6 +147,15 @@ class SimpleTextFieldController constructor(
         onValueChange(textFieldConfig.convertFromRaw(rawValue))
     }
 
+    override fun setOnChangeListener(
+        identifierSpec: IdentifierSpec,
+        listener: (IdentifierSpec, String) -> Unit
+    ) {
+        this.listener = {
+            listener(identifierSpec, it)
+        }
```

Comment:
Since we don’t use the `identifierSpec` in any way other than passing it back to the `listener`, can we instead do something like this at the call site?

```kotlin
// In SectionSingleFieldElement

override fun setTextFieldOnChangeListener(listener: (IdentifierSpec, String) -> Unit) {
  val onChangeListener: (String) -> Unit = { value ->
    listener(identifier, value)
  }
  controller.setOnChangeListener(onChangeListener)
}
```

This way, we can avoid passing the `IdentifierSpec` down.